src/pages/home/home.html

```diff
@@ -1,14 +1,37 @@
 <ion-header>
   <ion-navbar>
     <ion-title>
-      Ionic Blank
+      Tell Me About Yourself
     </ion-title>
   </ion-navbar>
 </ion-header>
 
 <ion-content padding>
-  The world is your oyster.
-  <p>
-    If you get lost, the <a href="http://ionicframework.com/docs/v2">docs</a> will be your guide.
-  </p>
-</ion-content>
+  <button ion-button block color='primary' (click)='add()'>
+    Add
+  </button>
+  <ion-list>
+    <ion-item *ngFor='let location of lists'>
```

Comment:
Is there a reason your loop is storing each user object as `location` ?

Reply:
No specific reason I was referencing a project I had recently and never changed the value. I will add all the other stuff in though, I have some user auth projects stashed away.